lib/inets/src/http_client/httpc.erl

```diff
@@ -58,6 +83,98 @@
 
 -define(DEFAULT_PROFILE, default).
 
+%%%=========================================================================
+%%%  Type specifications
+%%%=========================================================================
+
+%% These are used more than once in this module
```

Comment:
There are still no exported types from `httpc` or from any other modules in the `inets` app  I think. But if we should export types we must be very careful not to export to many types and we also have to be very careful with the names of the types. I think we have to go through `httpd`as well before we decide about exported types because there might be some types which should be common for the client and the server.